posts/packit_1_0_0_action_required/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Packit 1.0.0 countdown - action may be required"
+date: 2024-12-09T12:00:00+00:00
+authors: mmassari
+tags:
+  - configuration
+---
+
+After 105 minor releases we are almost ready to announce our first major release, Packit 1.0.0!
+
+<!--truncate-->
+
+## What's New
+
+We will **remove all long-deprecated keys from the configuration file**.
+
+`packit-service` jobs are tied with `packit CLI`. The _packit configuration code_ is defined in the `packit` package and used by the `packit-service`.
+So be aware that for those projects still using the deprecated keys in their `packit.yaml` configuration file we have created a migration pull request.
+
+Please, **check if you have any new PR for your project created by the packit-service bot account, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the new packit version is released.
+
+We plan to release Packit 1.0.0 in **January 2025**.
+
+### List of removed keys
+
+- Job type `build` has been changed to `copr_build`.
+- Job type `production_build` has been changed to `upstream_koji_build`.
```

Comment:
We can mention that this is for explicitness after introducing more build types.